storage/test.go

```diff
@@ -70,6 +71,7 @@ func NewTestStorageEngineInDir(t testing.TB, dir string) Engine {
 	t.Cleanup(func() {
 		_ = result.Shutdown()
 	})
+	fmt.Printf("Created test storage engine in %s\n", dir)
```

Comment:
You also get this information if you enable debug/trace logging on gorm.

Reply:
I knew I had seen it before, just didn't know how to activate it.
Since this is a test function, I would always like to know the path